src/Migrations/2022_03_28_073146_create_topic_table.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+use Uasoft\Badaso\Module\LMSModule\Helpers\DatabaseHelper;
+
+class CreateTopicTable extends Migration
+{
+    private $tableName;
+
+    public function __construct()
+    {
+        $this->tableName = DatabaseHelper::getBadasoTableName('topic');
+    }
+
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create($this->tableName, function (Blueprint $table) {
+            $table->id();
+            $table->text('title');
+            $table->foreignId('course_id');
+            $table->foreignId('created_by');
+            $table->timestamps();
+
+            $table->foreign('course_id')
+                ->references('id')
+                ->on(DatabaseHelper::getBadasoTableName('courses'))
+                ->onDelete('cascade')
+                ->onUpdate('cascade');
+            $table->foreign('created_by')
+                ->references('id')
+                ->on(DatabaseHelper::getBadasoTableName('users'))
+                ->onDelete('cascade')
+                ->onUpdate('cascade');
+        });
+    }
+
+    /**
+     * Reverse the migrations.
+     *
+     * @return void
+     */
+    public function down()
+    {
+        Schema::dropIfExists('topic');
```

Comment:
lol I guess I also did this mistake on my PR. We'll create the table "badaso_topic" by defining the `tableName` in the `__construct` function, so we need to drop the same table name

Reply:
And I guess we should use plural form for table name, CMIIW

Reply:
And can you please change my migration file as well, thanks a lot!

Reply:
ok omw